client/configuration_variable.go

```diff
@@ -77,6 +86,17 @@ type ConfigurationVariableUpdateParams struct {
 	Id           string
 }
 
+func (v ConfigurationVariable) MarshalJSON() ([]byte, error) {
+	v.Overwrites = nil
+
+	// This is done to prevent an infinite loop.
+	type ConfigurationVariableDummy ConfigurationVariable
```

Comment:
without doing the following an infinite loop will happen, because inside marshal we are calling marshal again on the same object.